core/domain/services/graphql/connection_resolvers/DatetimeConnectionResolver.php

```diff
@@ -131,9 +132,52 @@ public function get_query_args()
          * Merge the input_fields with the default query_args
          */
         if (! empty($input_fields)) {
-            $query_args = array_merge($query_args, $input_fields);
+            $where_params = array_merge($where_params, $input_fields);
         }
 
+        // ID of the offset datetime.
+        $offset = $this->get_offset();
+
+        /**
+         * Map the orderby inputArgs to the WP_Query
+         */
+        if (! empty($this->args['where']['orderby']) && is_array($this->args['where']['orderby'])) {
+            $query_args['order_by'] = [];
+            foreach ($this->args['where']['orderby'] as $orderby_input) {
+                $query_args['order_by'][ $orderby_input['field'] ] = $orderby_input['order'];
+            }
+        } elseif ($offset) {
+            $compare = ! empty($last) ? '<' : '>';
+            $where_params['DTT_ID'] = array($compare, $offset);
+        }
```

Comment:
nearly identical to lines 134 - 142 of `core/domain/services/graphql/connection_resolvers/TicketConnectionResolver.php`
Can be extracted and moved into `AbstractConnectionResolver`